README.md

```diff
@@ -20,9 +20,12 @@ From here on Dynatrace OneAgent Operator controls the lifecycle and keeps track
 
 ## Supported platforms
 
-Dynatrace OneAgent Operator is supported on the following platforms:
-* Kubernetes 1.9+
-* OpenShift Container Platform 3.9+
+Depending of the version of the Dynatrace OneAgent Operator, it supports the following platforms:
+
+| Dynatrace OneAgent Operator version | Kubernetes | OpenShift Container Platform |
+| ----------------------------------- | ---------- | ---------------------------- |
+| master                              | 1.11+      | 3.11+                        |
```

Comment:
v0.3.0+ / master
or something?

Reply:
I'll add an v0.3.0 row once we release it properly.